x/stake/client/cli/query.go

```diff
@@ -114,6 +114,76 @@ func GetCmdQueryValidators(storeName string, cdc *codec.Codec) *cobra.Command {
 	return cmd
 }
 
+// GetCmdQueryValidatorUnbondingDelegations implements the query all unbonding delegatations from a validator command.
```

Comment:
Do these use the queriers yet or directly to the store?

Reply:
They use queriers

Reply:
So shouldn't they use a `queryRoute` and not a `storeName`? Ref: https://github.com/cosmos/cosmos-sdk/pull/2565#discussion_r227646142

Reply:
ok so change it to `"stake"` instead of store name ?

Reply:
yes

Reply:
@alexanderbez addressed